.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
+  # Pre-reqs for building omsagent
+  - sudo apt-get install -qq pkg-config make ruby bison g++ rpm librpm-dev libpam0g-dev libssl-dev libmysqlclient-dev
+  - rvm use system                 # Use system ruby instead of ruby provided by rvm
```

Comment:
As per requirements with the upgrade to Ruby 2.3, this version must be at least 1.9. I believe an Ubuntu 14 build machine should pass with this, but I've asked Suju to double-check.